packages/website/ts/components/wallet/wallet.tsx

```diff
@@ -269,13 +274,31 @@ export class Wallet extends React.Component<WalletProps, WalletState> {
         });
     }
     private _renderFooterRows() {
-        const primaryText = '+ other tokens';
         return (
             <ListItem
                 key={FOOTER_ITEM_KEY}
-                primaryText={primaryText}
+                primaryText={
+                    <div className="flex">
+                        <FloatingActionButton mini={true} zDepth={0} onClick={this.props.onAddToken}>
+                            <ContentAdd />
+                        </FloatingActionButton>
+                        <FloatingActionButton mini={true} zDepth={0} className="px1" onClick={this.props.onRemoveToken}>
+                            <ContentRemove />
+                        </FloatingActionButton>
+                        <div
+                            style={{
+                                paddingLeft: 10,
+                                position: 'relative',
+                                top: '50%',
+                                transform: 'translateY(33%)',
+                            }}
+                        >
+                            add/remove tokens
```

Comment:
How would `+/- tokens` look? Don't feel strongly, but might be more compact.

Reply:
lemme iterate with ben!